packages/react/src/components/input/Input.tsx

```diff
@@ -243,7 +217,31 @@ export class Input extends Component<IInputProps, IInputComponentState> {
                 min={this.props.minimum}
                 max={this.props.maximum}
                 autoComplete="off"
-                {..._.omit(this.props, [...PropsToOmitUtils.input, ...inputPropsToOmit])}
+                {..._.omit(
```

Comment:
I think we're missing `children` and that's why the tests aren't happy

Reply:
removing the children from the regular props make all the tests pass. GG